### PR TITLE
 chore: write landing page for docs.fedimint.org 

### PR DESCRIPTION
**Pull request by dpc**
On top of https://github.com/fedimint/fedimint/pull/4624

Please point out nits, but approve anyway, as I need to see it published to improve further.

### CODE REVIEW

**Review by maan2003 — DISMISSED** [review]

**Review by maan2003 — APPROVED** [review]

File: docs/rustdoc-index.md [hunk 1]
```diff
@@ -1,5 +1,62 @@
...
 <!-- this page is used a landing page for https://docs.fedimint.org/ -->
+<!-- lots of links in this document are relative to the generated document, so disabling linkcheck altogether: -->
+<!-- markdown-link-check-disable -->
 
-This is documentation automatically generated from the Fedimint source code.
+## Target audience
+
+This documentation is automatically generated from the Fedimint source code,
+and it's meant for developers, builders and people who want to understand
+Fedimint project on an implementation level.
```
Comment (elsirion) [inline docs/rustdoc-index.md:11]: nit: "the Fedimint project"?

File: docs/rustdoc-index.md [hunk 2]
```diff
@@ -1,5 +1,62 @@
...
+* [Fedimint github](https://github.com/fedimint/fedimint)
+* [Fedimint chat](https://chat.fedimint.org/)
+
+## State
+
+This is a recently added document and subject of continuous change. Please report problems and submit improvements.
+
+# Overview
+
+Fedimint is implemented in [Rust](https://www.rust-lang.org/) and consists of multiple modules (called "crates" in Rust).
```
Comment (elsirion) [inline docs/rustdoc-index.md:25]: "modules" is a bit ambiguous in the Fedimint context, I didn't think you were talking about crates

File: docs/rustdoc-index.md [hunk 3]
```diff
@@ -1,5 +1,62 @@
...
+
+In fact core functions of Fedimint are implemented as modules:
+
+* [fedimint-mint-server](./fedimint_mint_server/index.html) and [fedimint-mint-client](./fedimint_mint_client/index.html) implement ecash.
+* [fedimint-wallet-server](./fedimint_wallet_server/index.html) and [fedimint-wallet-client](./fedimint_wallet_client/index.html) implement on-chain deposits and withdrawals.
+* [fedimint-ln-server](./fedimint_ln_server/index.html) and [fedimint-ln-client](./fedimint_ln_client/index.html) implement lightning integration.
+
+Some additional built-in modules are also available:
+
+* [fedimint-meta-server](./fedimint_meta_server/index.html) and [fedimint-meta-client](./fedimint_meta_client/index.html) implement guardian managing additional ("meta"/index.html) information about the Federation.
```
Comment (elsirion) [inline docs/rustdoc-index.md:45]: is `("meta"/index.html)` a typo? Could also link to meta field docs.

**Review by elsirion — APPROVED** [review]

**Comment by elsirion** [comment]
Why backport this? I don't think these docs will be built on docs.rs anyway. Maybe if we wanted to build docs for all versions one day?

**Pull request merged**